packages/task/src/browser/task-service.ts

```diff
@@ -205,7 +205,7 @@ export class TaskService implements TaskConfigurationClient {
             });
             const taskConfig = event.config;
             const taskIdentifier = taskConfig ? this.getTaskIdentifier(taskConfig) : event.taskId.toString();
-            this.messageService.info(`Task '${taskIdentifier}' has been started.`);
+            console.log(`Task '${taskIdentifier}' has been started.`);
```

Comment:
do we really need to see it in logs? Does task identifier has any meaningful information to track it back like in several days?

Reply:
i don't mind removing it if you don't see any value

Reply:
let's remove it, our logs already have a lot of noise, logging starting and stoping each task will make it even more noisy